app/models/transaction.rb

```diff
@@ -18,15 +20,10 @@ class Transaction < ApplicationRecord
   include BaseTransaction
   include TransactionHelpers
 
-  after_save    :recalculate_balances!
   after_destroy :recalculate_balances!
+  after_save    :recalculate_balances!
 
-  validates :id_at_client, presence: true, uniqueness: { scope: :issuer_id }, if: :is_client_transaction?
-
-  def signed_amount_for(user)
-    return -amount if user == debtor
-    return amount if user == creditor
-  end
```

Comment:
Ook dit lijkt mij geen rubocop ding?

Reply:
Dat klopt, het is unused code dus mag sowieso weg